### PR TITLE
Alter loop tables

### PR DESCRIPTION
**Pull request by Abhishek-lodha**
Migration added to add TESTING profile for loop user and to disable migration for ivr AUDIO model.

### CODE REVIEW

File: loop/views.py [hunk 1]
```diff
@@ -53,7 +54,7 @@ def dashboard(request):
 
 def filter_data(request):
     language = request.GET.get('language')
-    aggregators = LoopUser.objects.all().values('user__id', 'name', 'name_en')
+    aggregators = LoopUser.objects.filter(role=ROLE_AGGREGATOR).values('user__id', 'name', 'name_en')
```
Comment (lokeshgarg92) [inline loop/views.py]: Same thing needs to be handled at other places as well. E.g.  LINE 177 in function total_static_data(request)

File: loop/views.py [hunk 2]
```diff
@@ -173,7 +174,7 @@ def total_static_data(request):
         'farmer').annotate(farmer_count=Count('farmer')).exclude(farmer_count=1))
     total_farmers_reached = len(
         CombinedTransaction.objects.values('farmer').distinct())
-    total_cluster_reached = len(LoopUser.objects.all())
+    total_cluster_reached = len(LoopUser.objects.filter(role=ROLE_AGGREGATOR))
```
Comment (lokeshgarg92) [inline loop/views.py]: Are you sure, all cases are handled

File: loop/views.py (hunk 1 above)
Comment (Abhishek-lodha) [inline loop/views.py]: Done

File: loop/views.py (hunk 2 above)
Comment (Abhishek-lodha) [inline loop/views.py]: Yes

**Pull request merged**